source/_posts/vue-security-best-practices.md

```diff
@@ -393,7 +393,7 @@ Once you have some security headers configured properly check them with the [Sec
 Summary
 -------
 
-To begin with, Vue is a beautiful frontend framework. Its ecosystem and community are pretty good. The official documentation is your best source to deal with doubts, questions, and problems don't think too much to consult it. Before using any directive or feature of Vue check it the documentation (it offers functionality explanations, examples, and use cases). As we saw previously, "v-if" and "v-show" don't should be used indistinctly.
+To begin with, Vue is a beautiful frontend framework. Its ecosystem and community are pretty good. The official documentation is your best source to deal with doubts, questions, and problems don't think too much to consult it. Before using any directive or feature of Vue check it the documentation (it offers functionality explanations, examples, and use cases). As we saw previously, `v-if` and `v-show` don't should be used indistinctly.
```

Comment:
@cr0wg4n "`v-if` and `v-show` don't should be used indistinctly." => "should not be used"